config/ip/config.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ip
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+)
+
+// Configure configures ip group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_ip_group", func(r *config.Resource) {
+		r.Kind = "IPGroup"
```

Comment:
```suggestion
		r.Kind = "IPGroup"
		r.Group = "network"
```

I think we should follow [what SDK says](https://github.com/Azure/azure-sdk-for-go/blob/main/services/network/mgmt/2021-05-01/network/ipgroups.go#L1) for the grouping so that if we merge the providers, we don't have to handle GVK changes. You can check how they classify it by looking at the ID as well. For example, import ID in TF documentation looks like the following for this resource:
```
/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/ipGroups/myIpGroup
```

Which means it belongs to `Microsoft.Network` service, hence `network` group.

Reply:
Thank you @muvaf, that makes sense. Going over generated resources and checking their groups.